template/src/components/debug-menu/DebugMenu.js

```diff
@@ -0,0 +1,222 @@
+import BottomSheet, {BottomSheetFlatList} from '@gorhom/bottom-sheet'
+import React, {useCallback, useMemo, useRef, useState} from 'react'
+import {Modal, Pressable, SafeAreaView, ScrollView, View, Text, Button} from 'react-native'
+import {getApplicationName, getBuildNumber, getDeviceId} from 'react-native-device-info'
+import Draggable from 'react-native-draggable'
+
+import Config, {BOTTOM_SHEET_TYPE, CODEPUSH_KEYS, EXTRA_QA_ENVS} from '../../constants/configs'
+
+import {InfoMenu, InfoMenuRow, InfoMenuLink} from '../info-menu'
+
+import styles from './styles'
+
+import {colors} from '../../themes'
+import useWindowDimensions from 'react-native/Libraries/Utilities/useWindowDimensions'
```

Comment:
We should use `import { useWindowDimensions } from 'react-native'`

Reply:
Do we create another tịcket for this? we would like to merge localization